packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -853,6 +854,7 @@ export function TransferPanelMain({
     networks.destinationChain,
     isTestnetMode,
     setNetworks,
+    actions.app,
```

Comment:
would be better if we just kept this out so we don't introduce unnecessary rerenders

Reply:
actions.app is a stable reference, it doesn't cause rerender